grafana-cloud-metrics.yaml

```diff
@@ -304,6 +304,10 @@ resources:
             url=https://monitoring.api.rackspacecloud.com/v1.0
             EOL
             start graphite-api
+            ufw enable
+            ufw allow ssh
+            ufw allow http
+            ufw allow https
           params:
```

Comment:
As far as I'm aware, there isn't a better way to do this.  I don't see a problem with doing the firewall rules this way, anyway.